tests/external/iree-test-suites/onnx_ops/onnx_ops_gpu_rocm_rdna3.json

```diff
@@ -348,10 +348,6 @@
     "onnx/node/generated/test_lstm_with_peepholes",
     "onnx/node/generated/test_pow",
     "onnx/node/generated/test_pow_example",
-    "onnx/node/generated/test_pow_types_float32_int32",
-    "onnx/node/generated/test_pow_types_float32_int64",
-    "onnx/node/generated/test_pow_types_float32_uint32",
-    "onnx/node/generated/test_pow_types_float32_uint64",
```

Comment:
Yay, tests newly passing on other backends... but the diff between coverage on CPU and GPU is not good.